mergegi/mergegi.py

```diff
@@ -19,8 +32,14 @@ def mergegi(barcodes_csv, raw_data_dir, merged_data_dir, paired=False, merge_lan
     file_to_merge = defaultdict(lambda: defaultdict(lambda: defaultdict(list)))
     with open(barcodes_csv) as csvfile:
         csv_reader = csv.reader(csvfile, skipinitialspace=True)
-        # Skip header
-        next(csv_reader)
+        try:
+            # skip header and check if column count is good
+            samplename, barcode, barcode2, project, lane = next(csv_reader)
+        except ValueError:
+            msg = ("Samplesheet is malformed. You must have samples names, barcode1, barcode2,"
```

Comment:
Your sample sheet seems to be ill-formed. You must have 5 columns corresponding to: ....lanes. They are expected to be found in this order.